README.md

```diff
@@ -157,32 +134,62 @@ version:             0.0.0.0
 library
   exposed-modules:     Example
   build-depends:       base >= 4.10 && < 4.13
-                     , relude ^>= 0.4.0
+                     , relude ^>= 0.6.0.0
 
   mixins:              base hiding (Prelude)
                      , relude (Relude as Prelude)
 
   default-language:    Haskell2010
 ```
 
+> **NOTE:** if you use [`summoner`](https://github.com/kowainik/summoner) to generate Haskell project,
+> this tool can automatically create such structure for you when you specify custom prelude.
+
 If you want to use e.g. `Relude.Extra.Enum`, you need to list it
 (and potentially other modules, like `Relude.Unsafe`) under
 the `mixins` field as well, like this:
 
 ```cabal
-  mixins:              base hiding (Prelude)
-                     , relude (Relude as Prelude, Relude.Extra.Enum)
+  mixins: base hiding (Prelude)
+        , relude (Relude as Prelude, Relude.Extra.Enum)
 ```
 
-If you want to be able to import every module of relude, you need to add `relude`
-a second time, like this:
+If you want to bring into scope all `Extra.*` modules, you can add
+only `Relude.Extra` module to `mixins`, and later you can import all
+extra functions and data types from `Relude.Extra`. This is the
+easiest way to bring all safe functions from `relude` to your project.
 
 ```cabal
-  mixins:              base hiding (Prelude)
-                     , relude (Relude as Prelude)
-                     , relude
+  mixins: base hiding (Prelude)
+        , relude (Relude as Prelude, Relude.Extra)
 ```
 
+### base-noprelude [↑](#structure-of-this-tutorial)
+
+Alternatively, you can use the `base-noprelude` trick to use
+alternative preludes. This approach can be useful if you want to have
+your own `Prelude` with some custom functions, not provided by
+`relude`. To use the trick, perform the following steps:
+
+1. Replace the `base` dependency with corresponding version of `base-noprelude` in
+   your `.cabal` file.
+2. Add a `relude` dependency to your `.cabal` file.
+3. Add the following `Prelude` module to your project (both to filesystem and to `exposed-modules`):
```

Comment:
Let's separate this statement into 2 sentences, for example: Create the file ... Then add it to the `exposed-modules`...